client/src/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import '../index.css';
+import { accountRepository } from '@/_services';
+import { authenticationRepository } from '@/_services';
+import config from 'config';
+const signalR = require('@aspnet/signalr')
+
+class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            accountId: null,
```

Comment:
If the account object is used in more than one component, I'll make a model for it!